PRASFiles.jl/src/Results/utils.jl

```diff
@@ -0,0 +1,102 @@
+# Define function to get name of Type{PRAS.PRASCore.Results.Result}
+name(::Type{T}) where {T} = (isempty(T.parameters) ? T : T.name.wrapper)
+
+# PRAS SystemModel Type Parameters
+struct TypeParams
+    N::Int64
+    L::Int64
+    T::String
+    P::String
+    E::String
+end
+
+function TypeParams(pras_sys::SystemModel{N,L,T,P,E}) where {N,L,T,P,E}
+    return TypeParams(
+        N,
+        L,
+        unitsymbol(T),
+        unitsymbol(P),
+        unitsymbol(E),
+    )
+end
+
+# EUE Result
+struct EUEResult
+    mean::Float64
+    stderror::Float64
+end
+
+function EUEResult(shortfall::ShortfallResult; region::Union{Nothing, String} = nothing)
+    eue = 
+    if (region === nothing)
```

Comment:
I'm guessing the autoformatter is getting confused about assigning from if-blocks. Maybe use a ternary operator instead and have it all on one line?